pkg/dockerregistry/server/auth.go

```diff
@@ -125,14 +125,9 @@ func (ac *AccessController) Authorized(ctx context.Context, accessRecords ...reg
 		return nil, ac.wrapErr(err)
```

Comment:
I've modified getToken to fail if it doesn't actually get a valid token, and I added a unit test for it.